### PR TITLE
[14.0][IMP] account_due_list_payment: init module

### PR DESCRIPTION
**Pull request by felipemotter**
Hello guys,

This pull request introduces a new module that enables users to execute payments directly from the due_list view. This feature is designed with a focus on smaller businesses or companies that are not currently implementing bank reconciliation or need make the payment before this.

This new module allows payments to be made directly from the due_list view, simplifying the overall process.

### CODE REVIEW

**Review by antoniospneto — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Review by rvalyi — APPROVED** [review]

File: account_due_list_payment/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 Engenere.one
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account Due List Payment",
+    "summary": """
+        Allows you to make payments directly from the due list view""",
+    "version": "14.0.0.0.0",
```
Comment (rvalyi) [inline account_due_list_payment/__manifest__.py]: @felipemotter @antoniospneto the 1st version number is supposed to be 14.0.1.0.0, see https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#12version-numbers

other than that I would approve...

**Review by rvalyi — CHANGES_REQUESTED** [review]

File: account_due_list_payment/__manifest__.py (hunk 1 above)
Comment (antoniospneto) [reply account_due_list_payment/__manifest__.py]: @rvalyi Solved, thanks for noticing!

**Review by rvalyi — APPROVED** [review]

**Pull request merged**